tdrs-backend/docker-compose.yml

```diff
@@ -30,8 +30,8 @@ services:
     environment:
       - SERVICES=s3
       - DATA_DIR=/tmp/localstack/data
-      - AWS_BUCKET
-      - AWS_REGION_NAME
+      - AWS_BUCKET=tdp-datafiles-localstack
```

Comment:
Set these to a hardcoded value to prevent issues from not setting these locally and to allow us to remove them from Circle CI.

Reply:
Can you elaborate on why these can't be pulled from .env or an docker variable?

Reply:
It was before but not setting it would lead to the Django server failing to start. Since it doesn't matter what this value is as long as its the same in this container and in the Django settings (enforced in tdpservice.settings.local) it doesn't really provide any value for it to be configurable. Alternatively we could leave this configurable and just set a default so we don't run into the startup failure but it would have to be handled in all places that reference it - the localstack script (BASH), the wait-for-services script (also BASH) and the Django settings (python).